src/main/scala/no/ndla/draftapi/controller/DraftController.scala

```diff
@@ -64,9 +64,9 @@ trait DraftController {
     private val copiedTitleFlag =
       Param[Option[String]]("copied-title-postfix",
                             "Add a string to the title marking this article as a copy, defaults to 'true'.")
-    private val competences = Param[Option[String]](
-      "competences",
-      "Return only articles with one of the specified competence goals. Separate by comma to use specify multiple values (,).")
+    private val grepCodes = Param[Option[String]](
+      "grepCodes",
```

Comment:
Denne eksponeres som en query parameter. For å være konsekvent ville jeg nok heller ha holdt meg til kebab-case